File: low_load.cpp
#include <iostream>
#include <chrono>

int main() {

    long long sum = 0;
    long long n = 984600000LL;

    std::cout << "Program starting...\n";

    auto start = std::chrono::high_resolution_clock::now();
    std::cout << "Start time: "
              << std::chrono::duration<double>(start.time_since_epoch()).count()
              << "\n";

    for (long long i = 1; i <= n; i++) {
        sum += i;
    }

    auto end = std::chrono::high_resolution_clock::now();
    std::cout << "End time: "
              << std::chrono::duration<double>(end.time_since_epoch()).count()
              << "\n";

    double elapsed = std::chrono::duration<double>(end - start).count();
    std::cout << "Time elapsed: " << elapsed << " seconds\n";
    std::cout << "Sum: " << sum;

    return 0;
}

File: low_load.go
package main

import (
    "fmt"
    "time"
)

func main() {
    fmt.Println("Program starting...")

    start := time.Now()
    fmt.Printf("Start time: %.6f\n", float64(start.UnixNano())/1e9)

    sum := 0
    for i := 0; i < 984_600_000; i++ {
        sum += i
    }

    end := time.Now()
    fmt.Printf("End time: %.6f\n", float64(end.UnixNano())/1e9)

    elapsed := end.Sub(start).Seconds()
    fmt.Printf("Time elapsed: %.8f seconds\n", elapsed)
    fmt.Printf("Sum: %d\n", sum)
}
